presto-native-execution/presto_cpp/main/TaskResource.cpp

```diff
@@ -351,6 +359,7 @@ proxygen::RequestHandler* TaskResource::createOrUpdateTask(
             taskId,
             updateRequest,
             planFragment,
+            summarize,
```

Comment:
`deleteTask` also needs to support the flag: https://github.com/prestodb/presto/blob/5d63246c7dbd93cf0f6f444780bddbc05545bdda/presto-main/src/main/java/com/facebook/presto/server/TaskResource.java#L271

Reply:
Also
`TaskResource::getTaskInfo(`
and 
`public List<TaskInfo> getAllTaskInfo(`
Although the latter seems like not supported by native worker and might be only for coordinator?